src/Components/Search/SearchBar.tsx

```diff
@@ -411,7 +411,11 @@ export const SearchBarQueryRenderer: React.SFC = () => {
             }}
             render={({ props }) => {
               if (props) {
-                return <SearchBarRefetchContainer viewer={props.viewer} />
+                return (
+                  <Provider>
```

Comment:
I'm not 100% sure why this wound up working in storybooks (I couldn't find any provider being set in storybooks). This fixes the issue observed with `yarn link`, and storybooks still seems to work fine too!